frontend/src/components/common/camps/CampSessionInfoHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import { Container, Flex, IconButton, Text, Box } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronLeft,
+  faChevronRight,
+} from "@fortawesome/free-solid-svg-icons";
+
+import textStyles from "../../../theme/textStyles";
+
+type CampSessionInfoHeaderProps = {
+  camp: any;
+};
+
+const CampSessionInfoHeader = ({
+  camp,
+}: CampSessionInfoHeaderProps): React.ReactElement => {
+  const [currentCampSession, setCurrentCampSession] = useState(0);
+  const numSessions = camp.campSessions.length;
+
+  const nextSession = () => {
+    if (numSessions >= 2)
+      setCurrentCampSession(
+        currentCampSession === numSessions - 1 ? 0 : currentCampSession + 1,
+      );
```

Comment:
nit: we can use mod here to simplify things a bit!
```suggestion
      setCurrentCampSession((currentCampSession + 1 ) % numSession);
```